.github/workflows/deploy-contracts.yml

```diff
@@ -0,0 +1,97 @@
+name: "Deploy Contracts"
```

Comment:
I can kinda tell what is going on but can you elaborate on the bigger picture of how this is used in terms of all apps or when this is meant to be used?

Reply:
So it deploys the app's contracts to the specified network.  It is a manual git workflow that we can enter a wallet secret or key into for deployment.  Unfortunately you cannot use manual git workflows on non main/master branches so this workflow won't be able to be used until later.  But this is what we will use to deploy our contracts to a persistent test network (or maybe even a mainnet).  These are the contracts our deployed frontends will use.  I still have to write the code which deploys the frontend to gh pages, but it wouldn't be too useful without the deployed contracts.